spec/models/events/review_comment_spec.rb

```diff
@@ -26,6 +26,17 @@
 
 RSpec.describe Events::ReviewComment, type: :model do
   describe 'validations' do
+    subject { build :review_comment }
+
+    it 'is valid with valid attributes' do
+      expect(subject).to be_valid
+    end
+
+    it 'is not valid without github id' do
```

Comment:
This is a question. Are there any other significant fields values that might invalidate a ReviewComment, like an empty comment text?

Reply:
Yes! I should add that validation! Thanks!